opa-filter-autoconfigure/src/main/java/com/bisnode/opa/spring/security/filter/autoconfigure/OpaFilterConfiguration.java

```diff
@@ -4,25 +4,33 @@
 import org.springframework.context.annotation.Configuration;
 
 import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
 
 @Configuration
 @ConfigurationProperties(prefix = "opa.filter")
 class OpaFilterConfiguration {
     private String documentPath;
+    private List<String> whitelist = new ArrayList<>();
     private URI instance = URI.create("http://localhost:8181");
 
     OpaFilterConfiguration() {
     }
 
-    OpaFilterConfiguration(String documentPath, URI instance) {
+    OpaFilterConfiguration(String documentPath, List<String> whitelist, URI instance) {
```

Comment:
Without any Javadoc or other type of documentation, it's not obvious what exactly "whitelist" expects. Either name it "endpointsWhitelist" or something or add some documentation on what the whitelist is meant to contain.

Reply:
Good point, I will provide doc for this and rename the param.

Reply:
Updated param name and provided spring-configuration-metadata.json for all already existing properties.